deployment/hasura/migrations/Aerie/3_sequence_parcel/down.sql

```diff
@@ -0,0 +1,207 @@
+       inner join sequencing.parcel p on es.parcel_id = p.id;
+
+---
+
+alter table sequencing.expansion_set
+  alter column command_dict_id set not null;
+
+alter table sequencing.expansion_set
+  add constraint expansion_set_unique_name_per_dict_and_model
+    unique (mission_model_id, command_dict_id, name),
+
+  add foreign key (command_dict_id)
+    references sequencing.command_dictionary (id)
+    on delete cascade,
+
+  drop constraint expansion_set_unique_name_per_parcel_and_model,
+  drop constraint expansion_set_parcel_id_fkey;
+
+drop view sequencing.rule_expansion_set_view;
+
+create or replace  view sequencing.rule_expansion_set_view as
+select str.rule_id,
+       set.id,
+       set.name,
+       set.owner,
+       set.description,
+       set.command_dict_id,
+       set.mission_model_id,
+       set.created_at,
+       set.updated_at,
+       set.updated_by
+from sequencing.expansion_set_to_rule str left join sequencing.expansion_set set
+                                                    on str.set_id = set.id;
+
+
+alter table sequencing.expansion_set
+  drop column parcel_id;
+
+--------------------------------------
+---- Modify the User Sequence Table
+--------------------------------------
+alter table sequencing.user_sequence
+  add column authoring_command_dict_id integer,
+
+  add foreign key (authoring_command_dict_id)
+    references sequencing.command_dictionary
+    on delete cascade;
+
+
+comment on column sequencing.user_sequence.authoring_command_dict_id is e''
+  'Command dictionary the user sequence was created with.';
+
+--- Data Migration
+update sequencing.user_sequence
+set authoring_command_dict_id = p.command_dictionary_id
+from sequencing.user_sequence us
+       inner join sequencing.parcel p on us.parcel_id = p.id;
+
+alter table sequencing.user_sequence
+  alter column authoring_command_dict_id set not null;
+---
+
+alter table sequencing.user_sequence
+  drop constraint user_sequence_parcel_id_fkey,
+  drop column parcel_id,
+  drop column seq_json;
+
+-------------------------------
+------ Drop Parcel to Parameter Table
+-------------------------------
+drop table sequencing.parcel_to_parameter_dictionary;
+
+-------------------------------
+------ Drop Parameter Dictionary Table
+-------------------------------
+drop table sequencing.parameter_dictionary;
+
+--------------------------------------
+---- Drop the Seq Adaptation Table
+--------------------------------------
+alter table sequencing.parcel
+  drop constraint parcel_sequence_adaptation_id_fkey;
+drop table sequencing.sequence_adaptation;
+
+--------------------------------------
+---- Drop the Channel Dictionary Table
+--------------------------------------
+alter table sequencing.parcel
+  drop constraint parcel_channel_dictionary_id_fkey;
+drop table sequencing.channel_dictionary;
+
+--------------------------------------
+---- Drop the Parcel Table
+--------------------------------------
+drop table sequencing.parcel;
```

Comment:
This block should be moved to above L183. Most importantly, this would make the `down` and the `up` consistent, but this would also avoid the need to alter this table on L186 and L193